src/objects/Customizable_Rollup_Settings__c.object

```diff
@@ -200,11 +200,11 @@
         <unique>false</unique>
     </fields>
     <fields>
-        <fullName>Rollups_Contact_HardCredit_ChunkSize__c</fullName>
+        <fullName>ChunkSize_Contact_HardCredit__c</fullName>
         <description>Only used by the new Customizable Rollups engine in LDV Orgs and only when required for LDV orgs</description>
         <externalId>false</externalId>
-        <inlineHelpText>LDV ORGS ONLY: If chunking is needed to avoid a Query Timeout in the batch job, this specifies the number of Contacts processed in each batch.</inlineHelpText>
-        <label>Contact Hard Credit Chunk Size</label>
+        <inlineHelpText>FOR LDV ORGS ONLY WITH SFDO GUIDANCE. To avoid a Query Timeout in the rollup batch job in orgs with many millions of Contacts, this specifies the maximum number of Contacts processed in each batch.</inlineHelpText>
+        <label>LDV: Contact Hard Credit Chunk Size</label>
         <precision>7</precision>
```

Comment:
@kslewis One more change. The precision on these three fields be set to 8 instead of 7 to allow for a number over 10,000,000.

Reply:
Ok, updated!